ts/features/fastLogin/store/selectors/index.ts

```diff
@@ -20,18 +32,32 @@ export const isFastLoginEnabledSelector = createSelector(
     )
 );
 
-export const fastLoginSelector = (state: GlobalState) =>
-  state.features.loginFeatures.fastLogin;
+/**
+ * return the remote config about FastLogin enabled/disabled
+ * based on a minumum version of the app, combined with FL opt-in user preference.
+ * If there is no data from backed nor user expressed an optin preference,
+ * false is the default value -> (FastLogin disabled)
+ */
+export const isFastLoginEnabledSelector = createSelector(
+  isFastLoginFFEnabled,
+  isFastLoginOptinEnabledSelector,
+  (fastloginFFEnabled, optInEnabled) => fastloginFFEnabled && !!optInEnabled
+);
```

Comment:
As above, about having two separates pairs (Remote/Local), FF.

Reply:
```mermaid
graph TD
A[App IO Open] -->|fastLoginEnabled = true| B[fastLoginEnabled?]
B -->|optInEnabled = true| C[optInEnabled?]
B -->|optInEnabled = false| D[Fast Login Only]
C --> E[Show Complimentary Screen]
E -->|User Chooses Fast Login| F[Fast Login]
E -->|User Chooses Standard Login| G[Standard Login]
A -->|fastLoginEnabled = false| H[Standard Login Only]
```

Reply:
If we will never release fast login without opt-in (since it's a core feature), why should we split the two features?

Reply:
To cover the future scenario where we could remove the opt-in, leaving only the FL feature. Wasn't it a possible scenario? If not, I agree with you that we can leave things the way they are.

Reply:
I'll find out and let you know.